frontend/src/bundles/studio/helpers/get-total-time.ts

```diff
@@ -0,0 +1,13 @@
+const calculateTotalMilliseconds = (
+    scripts: { duration: number }[],
```

Comment:
We have a separate type for Script, you can try using it

Reply:
To be bale to use the helper function in scripts and scenes I cant use the script type.